src/main/java/org/apache/commons/compress/archivers/zip/ZipFile.java

```diff
@@ -374,6 +374,8 @@ private ZipFile(final SeekableByteChannel channel, final String archiveName,
             }
             fillNameMap();
             success = true;
+        } catch (IOException e) {
+            throw new IOException("Error on ZipFile " + archiveName + e.getMessage(), e);
```

Comment:
add "; " before `+ e.getMessage()` or drop the message completely.

Reply:
dropped the message